RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Interface.swift

```diff
@@ -307,69 +287,3 @@ extension FactorSourcesClient {
 		return factorSource
 	}
 }
-
-// Move elsewhere?
-extension MnemonicWithPassphrase {
```

Comment:
moved to `Stage1MigrateToSargon+MnemonicWithPassphrase`: https://github.com/radixdlt/babylon-wallet-ios/blob/sargon_profile_integration_stage_1_batch_1/RadixWallet/ProfileSargonMigration/Stage1/Stage1MigrateToSargon%2BMnemonicWithPassphrase.swift#L7 with a primary implementation moved into Sargon :)